src/components/BAccordion/BAccordion.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <section
+    :id="id"
+    role="heading"
```

Comment:
Last structural change, I see what I was missing about where the heading role should go and why you put it on the button. The section should contain both a heading and a region. The heading should then contain the button.